### PR TITLE
Updated tokens related information (DOC-29)

### PR DESCRIPTION
**Pull request by Vineet-nuralogix**
- Updated tokens related information
- Some language edits

### CODE REVIEW

**Review by sambhare — APPROVED** [review]

File: apiary.apib [hunk 1]
```diff
@@ -763,10 +785,10 @@ account. The rest of the fields are optional.
 ### Login [POST /users/auth]
 
 Logs a user into a new session using Email, Password and optionally an MFAToken
-and responds with a User Token. The Token must be passed to every subsequent
-API call to the server.
+and responds with a User Token Pair(a User Token and a One-Time Refresh Token).
```
Comment (sambhare) [inline apiary.apib:788]: Need a space between `Pair` and `(a`

File: apiary.apib [hunk 2]
```diff
@@ -1646,7 +1668,7 @@ provided in error message.
 
 Provides a historical list of measurements captured by the API store. The
 results of the measurements are captured and only displayed for the current
-application providers token designator. Each record has a status representing
+application provider's token designator. Each record has a status representing
```
[the remark this answers is not in the document]
Comment (sambhare) [reply apiary.apib:1671]: If you mean the apostrophe in `provider's` we do need it

File: apiary.apib [hunk 3]
```diff
@@ -2078,13 +2100,14 @@ logged in as a DFX_LEAD,  DFX_ORG_ADMIN
 
 ### Register License [POST /organizations/licenses]
 
-Allows consumers to exchange a license key for a device token.
+Allows consumers to exchange a license key for a device token pair(a Device token
```
[the remark this answers is not in the document]
Comment (sambhare) [reply apiary.apib:2103]: Also need a space between `pair` and `(a`

File: apiary.apib [hunk 4]
```diff
@@ -2098,7 +2121,7 @@ communication with DeepAffex. When set, the server will compare the token value
 with HTTP request's `Referer` header and reject the request if those don't
 match.
 
-**Note:** Please make sure to store the obtained token locally as most
+**Note:** Please make sure to store the obtained tokens locally as most
```
Comment (sambhare) [inline apiary.apib:2124]: I think we should replace the word `locally` with `securely` - the storage doesn't need to be local but it does need to be secure.

**Review by daphnezhou — APPROVED** [review]

**Review by hiradkarimi — APPROVED** [review]

**Pull request merged**